### PR TITLE
Add support of commands `quit`, `scan`, `monitor` and `rpoplpush`  for redisMock

### PR DESCRIPTION
**Pull request by DocRoms**
Because I want to add few command on the REDIS mock 

* [x] Add `quit` command
* [x] Add `rpolpush` method
* [x] Add `scan` command
* [x] Add `monitor` command
* [x] Add tests

### CODE REVIEW

File: src/M6Web/Component/RedisMock/RedisMock.php [hunk 1]
```diff
@@ -1079,4 +1079,41 @@ protected function deleteOnTtlExpired($key)
...
+        return;
+    }
+
+    /**
+     * Mock the `scan` command
+     * @see https://redis.io/commands/scan
+     */
+    public function scan(): array
+    {
+        return [];
```
Comment (fabdsp) [inline src/M6Web/Component/RedisMock/RedisMock.php]: If you return a empty array, how can you use the mock ? You should scan the dataValues and look into deleteOnTtlExpired.

File: src/M6Web/Component/RedisMock/RedisMock.php [hunk 2]
```diff
@@ -1079,4 +1079,41 @@ protected function deleteOnTtlExpired($key)
...
+    }
+
+    /**
+     * Mock the getProfile()
+     */
+    public function getProfile()
```
Comment (fabdsp) [inline src/M6Web/Component/RedisMock/RedisMock.php]: Why getProfile ? is it a redis command ? Why do you return a ProfileMock ?

File: src/M6Web/Component/RedisMock/RedisMock.php [hunk 3]
```diff
@@ -1079,4 +1079,83 @@ protected function deleteOnTtlExpired($key)
...
+        for($i = $cursor; $i <= $count; $i++)
+        {
+            if (isset($keysArray[$i])){
+                $nextCursorPosition = $i;
+
+                if ('*' === $match || 1 === preg_match($pattern, $keysArray[$i])){
+                    $values[] = $keysArray[$i];
+                }
+
+            }else{
```
Comment (fabdsp) [inline src/M6Web/Component/RedisMock/RedisMock.php]: cs fixer

File: README.md [hunk 4]
```diff
@@ -69,6 +69,7 @@ Redis command                                    | Description
 **LRANGE** *key* *start* *stop*                  | Gets a range of elements from a list
 **MGET** *array\<field\>*                        | Gets the values of multiple keys
 **MSET** *array\<field, value\>*                 | Sets the string values of multiple keys
+**QUIT**                                         | Quit the REDIS
```
Comment (pabedu) [inline README.md]: https://www.youtube.com/watch?v=lLuc6rtWkrM&feature=youtu.be&t=93

**Pull request merged**